exercises/shared/.docs/tests.md

```diff
@@ -0,0 +1,42 @@
+# Tests
+
+## Setup
+
+Go through the setup [instructions for JavaScript][docs-exercism-javascript] to install the necessary dependencies.
```

Comment:
The link seems to be missing. Was this intended?

Reply:
Not intended. I missed it while moving this line from `cli.md`. Thanks for spotting :+1: